pkg/apis/autoscaling/types.go

```diff
@@ -96,20 +96,31 @@ type HorizontalPodAutoscalerSpec struct {
 	// +optional
 	Metrics []MetricSpec
 
-	// Behavior contains settings to specify the scaling behavior both while scaling up
-	// and scaling down. It also is used to customize other aspects of scaling like the
-	// scale down stabilization window
-	// +option
+	// Behaviour contains the scaling behavior for the HPA
```

Comment:
The comment for the behavior field should contain information about what the behavior field is a wrapper for. In this case it is the top level field to contain all the values which customize how the controller scales the target. Specific are not necessary at this stage because they can be explained in the comments of individual fields. Similar the defaults here need not be specified in full detail because each of the fields has it's own defaults.

Reply:
@thockin explicitly asked me to add information about default values for any non-specified field. And I'm not sure how to do that properly:
- If I don't specify the default `behavior` here, it might confuse the reader. 
- If I specify it here, it means, that I should explain all underlying abstractions at this point (`HPAScalingRules`, `HPAScalingPolicy`, etc)
- Moreover, in any case, I will have several places where I put the same "default values" (i.e. [behavior usage](https://github.com/kubernetes/kubernetes/pull/74525/files?file-filters%5B%5D=.go&file-filters%5B%5D=.json&file-filters%5B%5D=.pb&file-filters%5B%5D=.proto&file-filters%5B%5D=.yaml#diff-e9b775adb394bd5cd6c7606650443d57R102) and [behavior declaration](https://github.com/kubernetes/kubernetes/pull/74525/files?file-filters%5B%5D=.go&file-filters%5B%5D=.json&file-filters%5B%5D=.pb&file-filters%5B%5D=.proto&file-filters%5B%5D=.yaml#diff-e9b775adb394bd5cd6c7606650443d57R116) )
@thockin : could you give an advice?

Reply:
I've updated the comment